sdk/storage/azblob/internal/testcommon/clients_auth.go

```diff
@@ -57,6 +57,12 @@ const (
 	FakeStorageURL     = "https://fakestorage.blob.core.windows.net"
 )
 
+var (
+	CoolAccessTier    = blob.AccessTierCool
```

Comment:
Do we need to redefine here again for testing? we could have used the predefined types itself.

Reply:
Updated! Was having issues with this, but I've made it a local variable which works fine.